linkis-dist/package/conf/linkis.properties

```diff
@@ -100,4 +100,12 @@ wds.linkis.errorcode.auth.token=BML-AUTH
 wds.linkis.client.test.common.tokenValue=LINKIS_CLI_TEST
 wds.linkis.filesystem.token.value=WS-AUTH
 wds.linkis.gateway.access.token=WS-AUTH
-wds.linkis.server.dsm.auth.token.value=DSM-AUTH
\ No newline at end of file
+wds.linkis.server.dsm.auth.token.value=DSM-AUTH
+
+
+# s3 file system
+wds.linkis.storage.s3.access.key=
+wds.linkis.storage.s3.secret.key=
+wds.linkis.storage.s3.endpoint=
+wds.linkis.storage.s3.region=
+wds.linkis.storage.s3.bucket=
```

Comment:
remove the `wds.` prefix

Reply:
Thanks for your advice.I've dropped the prefix.